### PR TITLE
Scalable History

### PR DESCRIPTION
**Pull request by qti3e**
This PR intends to create a scalable transaction history for the XTC canister.

### CODE REVIEW

File: candid/xtc-history-e2e.did [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    Burn     : record {
+        from : principal;
+        to   : principal;
+    };
+    CanisterCalled : record {
+        canister : principal;
+        method_name: text;
+    };
+    CanisterCreated : record {
+        canister : principal;
```
Comment (kanej) [inline candid/xtc-history-e2e.did:21]: This will need the from address

**Comment by kanej** [comment]
@qti3e can you add a description of how the history canister works in the PR to aid reviewers

File: candid/xtc-history-e2e.did (hunk 1 above)
Comment (qti3e) [reply candid/xtc-history-e2e.did:21]: That change should not be in the scope of this PR. It should be handled in another PR, either after merging this one or before.

**Review by kanej — APPROVED** [review]
LGTM

**Pull request merged**